core/adminserver/admin_server.go

```diff
@@ -195,11 +188,9 @@ func treeConfig(treeTemplate *tpb.CreateTreeRequest, privKey *any.Any, directory
 	}
 
 	config.Tree.Description = fmt.Sprintf("KT directory %s", directoryID)
-	maxDisplayNameLen := 20
```

Comment:
I try to avoid magic numbers. Perhaps this should be moved into a package constant?